ironfish/src/rpc/routes/peer/addPeer.ts

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { DEFAULT_WEBSOCKET_PORT } from '../../../fileStores/config'
+import { ApiNamespace, router } from '../router'
+
+export type AddPeerRequest = {
+  host: string
+  port?: number
+  whitelist?: boolean
+}
+
+export type AddPeerResponse = {
+  added: boolean
+  error?: string
```

Comment:
Is this used? I guess it's nice to have in the future, but seems like we can just add this field when we implement it.

Reply:
Yea agreed, I'll remove